res/values/strings.xml

```diff
@@ -12246,6 +12246,8 @@
     <string name="connectivity_check_title">Internet connectivity check</string>
     <string name="connectivity_check_summary">HTTP endpoints to use for performing internet connectivity checks.</string>
     <string name="deny_new_usb_title">USB accessories</string>
+    <string name="auto_reboot_title">Auto reboot</string>
+    <string name="auto_reboot_summary">Automatically reboot the device, if no correct password has been provided under select number of hours.</string>
```

Comment:
Password here is too specific; there are other ways someone can unlock their phone (fingerprint, face, pattern, or PIN). It would be better to say something like "if the phone hasn't been unlocked in the selected number of hours"

Reply:
[done.](https://github.com/GrapheneOS/platform_packages_apps_Settings/pull/38/commits/88ae4dec952fd0433768364c3a373209457b9b1f)